pallets/settlement/src/lib.rs

```diff
@@ -586,6 +589,27 @@ decl_module! {
             Ok(())
         }
 
+        /// Edit a venue's signers.
+        /// * `id` specifies the ID of the venue to edit.
+        /// * `signers` specifies the signers to add/remove.
+        /// * `update_type` specifies the update type add/remove of venue.
+        #[weight = <T as Config>::WeightInfo::update_venue_signers(signers.len() as u32)]
+        pub fn update_venue_signers(origin, id: VenueId, signers: Vec<T::AccountId>, update_type: bool) -> DispatchResult {
```

Comment:
Only add extrinsics after the existing ones (end of the `decl_module!` block).

Reply:
Also move most of the extrinsics logic into a `base_update_venue_signers` function like is done with the `add_instruction` extrinsic.  We do this because the Code coverage tools don't work well with the `decl_module!` macro.